server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestNodesHotThreadsAction.java

```diff
@@ -34,23 +34,64 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Locale;
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-
 public class RestNodesHotThreadsAction extends BaseRestHandler {
 
+    private static final String formatDeprecatedMessageWithoutNodeID = "[%s] is a deprecated endpoint. " + 
+        "Please use [/nodes/hot_threads] instead.";
```

Comment:
Sorry, I just noticed this should say "/_nodes" and not "/nodes". Same for the next line.

Reply:
No problem! Thanks.